app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -331,6 +331,11 @@ class Settings private constructor(
         default = false
     )
 
+    val useStandardTrackingProtection by booleanPreference(
```

Comment:
Oh, so did we not have this before?

It's a little weird that this isn't a radio preference...in theory does that mean that the settings could get messed up and more than one of these could be selected? 🤔 (out of scope for this bug I know)

Reply:
Yeah you could just use this logic without adding the new setting https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/components/TrackingProtectionPolicyFactory.kt#L30-L33